homeassistant/components/energyzero/const.py

```diff
@@ -14,3 +16,13 @@
     "today_energy": "Energy market price",
     "today_gas": "Gas market price",
 }
+SERVICE_NAME: Final = "get_prices"
+SERVICE_PRICE_TYPES: Final = ["energy", "gas", "all"]
+SERVICE_SCHEMA: Final = vol.Schema(
+    {
+        vol.Required("type"): vol.In(SERVICE_PRICE_TYPES),
+        vol.Optional("start"): str,
+        vol.Optional("end"): str,
+        vol.Optional("incl_btw", default=False): bool,
```

Comment:
```suggestion
        vol.Optional("incl_btw", default=True): bool,
```

The library sets it default to True and we should keep it for constistency.